app/workers/notifications/create_date_alerts_notifications_job.rb

```diff
@@ -28,6 +28,17 @@
 
 module Notifications
   class CreateDateAlertsNotificationsJob < ApplicationJob
+    include GoodJob::ActiveJobExtensions::Concurrency
+
+    good_job_control_concurrency_with(
+      enqueue_limit: 1,
+      perform_limit: 1
+    )
+
+    retry_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError,
+             wait: 5.minutes,
+             attempts: 3
```

Comment:
For this job, I think a `total_limit` of 1 makes sense. With that in place, we don't need the `retry_on` (at least for the `ConcurrencyExceededError`)